docs/io/output/rpacket_tracking.ipynb

```diff
@@ -43,12 +43,7 @@
    "id": "4b0de6ca",
```

Comment:
<p>Before accessing a packet at a particular index, I'd advise to show the entire multi-indexed dataframe - just put <code>sim.runner.rpacket_tracker_df.</code> in  cell above </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>